modules/gitops-argo-cd-notification.adoc

```diff
@@ -2,23 +2,54 @@
 //
 // * argocd_instance/argo-cd-cr-component-properties.adoc
 
-:_content-type: PROCEDURE
+:_mod-docs-content-type: PROCEDURE
 [id="gitops-argo-cd-notification_{context}"]
 = Enabling notifications with Argo CD instance
 
-To enable or disable the link:https://argo-cd.readthedocs.io/en/stable/operator-manual/notifications/[Argo CD notifications controller], set a parameter in the Argo CD custom resource. By default, notifications are disabled. To enable notifications, set the `enabled` parameter to `true` in the `.yaml` file:
+Argo CD notifications allow you to send notifications to external services when events occur in your Argo CD instance. For example, you can send notifications to Slack or email when a sync operation fails. By default, notifications are disabled in Argo CD instances.
+
+.Prerequisites
+* You have access to the cluster with `cluster-admin` privileges.
+* You have access to the {OCP} web console.
```

Comment:
You could honestly combine these 2 bullets to be 
```suggestion
* You have access to an {OCP} cluster with `cluster-admin` privileges and are logged into the web console.
```

Reply:
Addressed. Thank you.